unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/Raycast/Serializer/Raycast.cs

```diff
@@ -0,0 +1,22 @@
+﻿using DCL.ECSComponents;
+using Google.Protobuf;
+
+namespace DCLPlugins.ECS7.ECSComponents.Raycast.Serializer
+{
+    public static class RaycastSerializer
+    {
+        public static byte[] Serialize(PBRaycast model)
+        {
+            int size = model.CalculateSize();
+            byte[] buffer = new byte[size];
+            CodedOutputStream output = new CodedOutputStream(buffer);
+            model.WriteTo(output);
+            return buffer;
+        }
+
+        public static PBRaycast Deserialize(object data)
+        {
+            return PBRaycast.Parser.ParseFrom((byte[])data);
+        }
+    }
+}
```

Comment:
I've recently added a generic serializer for protobuf, so we can avoid creating a custom for each component (unless needed) here: https://github.com/decentraland/unity-renderer/blob/dev/unity-renderer/Assets/DCLPlugins/ECS7/ProtocolBuffers/Serialization/ProtoSerialization.cs
usage at MaterialRegister as example: https://github.com/decentraland/unity-renderer/blob/dev/unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/Material/MaterialRegister.cs

Reply:
thanks!